src/applications/mhv/medical-records/components/RecordList/RecordListItem.jsx

```diff
@@ -11,6 +12,14 @@ import AllergyListItem from './AllergyListItem';
 const RecordListItem = props => {
   const { record, type } = props;
 
+  const history = useHistory();
+
+  useEffect(() => {
+    return history.listen(() => {
+      window.location.reload();
+    });
+  }, []);
+
```

Comment:
This looks like it will reload the page when you unmount this component. Is this a holdover from your previous ticket?